legal-api/report-templates/continuationApplication.html

```diff
@@ -24,29 +24,28 @@
          </table>
          <div class="business-name-header">
             <label class="lbl-business-name">
-            {% if nameRequest.legalName %}
-               {{ nameRequest.legalName }}
-            {% else %}
-               {{ numberedDescription }}
-            {% endif %}
+               {% if business.legalName %}
+                  {{ business.legalName }}
+               {% else %}
+                  {{numberedDescription}}
+          {% endif %}
```

Comment:
You still need this. 
Once business created you will have named/numbered legal name in `business.legalName`.
If business not create and it a named company `nameRequest.legalName`
If business not create and it a numbered company `numberedDescription`
```suggestion
               {% if business.legalName %}
                  {{ business.legalName }}
               {% elif nameRequest.legalName %}
                  {{ nameRequest.legalName }}
               {% else %}
                  {{numberedDescription}}
               {% endif %}
```

Reply:
Thanks, will update

Reply:
while updating this, can you update here as well https://github.com/bcgov/lear/blob/main/legal-api/report-templates/incorporationApplication.html#L27

Reply:
Sure, just updated